test/e2e/alertmanager_test.go

```diff
@@ -231,3 +226,187 @@ receivers:
 		t.Fatal(err)
 	}
 }
+
+func TestAlertmanagerZeroDowntimeRollingDeployment(t *testing.T) {
+	t.Parallel()
+
+	ctx := framework.NewTestCtx(t)
+	defer ctx.Cleanup(t)
+	ns := ctx.CreateNamespace(t, framework.KubeClient)
```

Comment:
This is the reason the tests are failing right now. The Prometheus instance can not be started, because the Prometheus service account is not present in the test namespace.